pulp_ansible/app/serializers.py

```diff
@@ -648,11 +811,51 @@ class CollectionVersionSerializer(ContentChecksumSerializer, CollectionVersionUp
 
     creating = True
 
+    def create(self, validated_data):
+        artifact = validated_data.get("artifact")
+        repository = validated_data.get("repository")
```

Comment:
This could technically return `None` which will result in an attribute error on the next line.